flocker/acceptance/testtools.py

```diff
@@ -203,6 +203,61 @@ def _clean_node(test_case, node):
         pass
 
 
+def get_volume_backend(test_case):
+    """
+    Get the volume backend the acceptance tests are running as.
+
+    :param test_case: The ``TestCase`` running this unit test.
+
+    :raise SkipTest: if the backend is specified.
+    """
+    backend = environ.get("FLOCKER_ACCEPTANCE_VOLUME_BACKEND")
+    if backend is None:
+        raise SkipTest(
+            "Set acceptance testing volume backend using the " +
+            "FLOCKER_ACCEPTANCE_VOLUME_BACKEND environment variable.")
+    return backend
```

Comment:
Let's make non-strings the default for representing things like that.  Doesn't have to be `twisted.python.constants` but should at least be something in that ballpark.